next/components/AppLayout/Navigation/SubnavigationColumn.tsx

```diff
@@ -50,11 +50,14 @@ const Column = ({ section, latestEvents, classNames }: ColumnProps) => {
                   {latestEvents.map((event) => (
                     <div key={event.attributes?.slug}>
                       <div className="h-23 w-[380px] cursor-pointer py-5">
-                        <NavigationMenu.Link className="h-10 pt-4 text-foreground-body" tabIndex={-1}>
+                        <NavigationMenu.Link
+                          className="h-10 pt-4 text-foreground-body"
+                          tabIndex={-1}
+                        >
                           <Link href={`/${event.attributes?.slug}`} passHref>
                             <a href={`/${event.attributes?.slug}`} className="flex">
                               <div className="flex h-16 w-20 bg-promo-yellow text-center">
-                                <DateCardDisplay
+                                <EventDetailsDateBox
```

Comment:
The text color of date should be darker as everywhere else.
![image](https://user-images.githubusercontent.com/19418224/210179453-12730252-6776-4a27-b256-8962379d56e1.png)

Reply:
Fixed.